x-pack/plugins/ingest_manager/server/plugin.ts

```diff
@@ -152,6 +157,35 @@ export class IngestManagerPlugin
   }
 
   public async setup(core: CoreSetup, deps: IngestManagerSetupDeps) {
+    const maxConcurrentRequests = 1;
```

Comment:
All this logic was put here out of laziness, it should really be placed in another file...